### PR TITLE
password generator fix

### PR DESCRIPTION
**Pull request by ltrilety**
Partially fix https://github.com/Tendrl/api/issues/117
The password is not longer, but much more complex as there is much more available characters. In other words from about 4E9 combinations to 1E15 combinations.
Users still don't have the option to choose a password.

### CODE REVIEW

**Comment by japplewhite** [comment]
I would really prefer a process where admin password is set to admin and then the user has to reset it in the UI upon logging in with a password of "admin".

I guess this is an enhancement @julienlim can we work up a UI screen for this?
Competing tools do this and it's easier for the end user.

**Comment by julienlim** [comment]
@japplewhite @ltrilety If admin password is set to admin is part of the installer flow, i.e. it should also be the place whee the admin password is set (vs. in the UI).  While it can be done in the UI, it seems like it should happen right there and then at the end of the installer workflow.

Thoughts?

**Comment by japplewhite** [comment]
@julienlim my suggestion was not to change it to admin in the ui but only to capture when someone logs in with admin/admin and force them to update the password. This would require a password management screen.
 To support this I'd prefer if we not introduce complex passwords up front since we will have divergent ways of installing Tendrl - ansible, qcow image, etc. so having admin simplifies user experience and forcing the password change should keep the security aspect acceptable.

File: Rakefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ namespace :etcd do
   task :load_admin do
     p 'Generating default Tendrl admin'
     etcd_config = Tendrl.etcd_config(ENV['RACK_ENV'])
-    password = SecureRandom.hex(4)
+    password = SecureRandom.base64(8)
```
Comment (japplewhite) [inline Rakefile]: I'd prefer to simply set it to admin and force a password change on login to the UI

**Comment by anivargi** [comment]
@japplewhite it is set to "adminuser" now as default password.
There is a API to update the user password via the API, but this needs to be implemented on the UI.

**Comment by julienlim** [comment]
@japplewhite @gnehapk @cloudbehl @nthomas-redhat @brainfunked @r0h4n @nnDarshan @mcarrano 

UI design for setting the "admin" user password may be seen at: https://redhat.invisionapp.com/share/6T900V2ZX.  Note: In the design the user is "Admin" but depending on what the actual value is, please update according to the correct upper/lower case.

**Pull request merged**